src/BetterReflection.php

```diff
@@ -84,4 +92,13 @@ public function findReflectionsOnLine() : FindReflectionOnLine
         return $this->findReflectionOnLine
             ?? $this->findReflectionOnLine = new FindReflectionOnLine($this->sourceLocator(), $this->astLocator());
     }
+
+    public function sourceStubber() : SourceStubber
```

Comment:
Possibly `private`?

Reply:
1. It's public because of tests.
2. All other methods are public too.

Reply:
Works for me :+1: